cloudant-sync-datastore-android/src/test/java/com/cloudant/sync/replication/ReplicationServiceTest.java

```diff
@@ -169,17 +175,24 @@ public void operationStarted(int operationId) {
         service.setReplicators(mMockReplicators);
         try {
             assertTrue("The countdown should reach zero", latch.await(DEFAULT_WAIT_SECONDS, TimeUnit.SECONDS));
-            verify(mMockPreferencesEditor, never()).putLong(captorPrefKeys.capture(), captorPrefValues.capture());
+            verify(mMockPreferencesEditor, times(2)).putLong(captorPrefKeys.capture(), captorPrefValues.capture());
+            List<String> prefsKeys = captorPrefKeys.getAllValues();
+            List<Long> prefsValues = captorPrefValues.getAllValues();
+            long expectedElapsedTime = SystemClock.elapsedRealtime() - (1000 * service.getUnboundIntervalInSeconds() / 2);
```

Comment:
Although you've written some clear comments for each test I'm struggling to understand what these magic numbers like `/ 2` are for. Perhaps pulling them out into constants with comments might help.